openage/launcher/mod.py

```diff
@@ -0,0 +1,64 @@
+import toml
+
+
+class ModInfo:
+    """
+    Parses and exposes info about a mod
+    """
+    def __init__(self, file_path=None, url=None):
+        if file_path is not None:
+            with open(file_path, 'r') as f:
+                info = toml.load(f)
+        elif url is not None:
+            info_str = some_black_magic(url)    # TODO: pseudocode
+            info = toml.loads(info_str)
```

Comment:
Modpacks should never be loaded by URL alone. They should always come from a repo. If we do this, we open ourselves for all kinds of security attacks.